### PR TITLE
aadii0408 patch 1

### PR DESCRIPTION
**Pull request by aadii0408**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4741"></a>

### CODE REVIEW

**Review by kumarsonsoff3 — DISMISSED** [review]
You've merge conflicts here because your fork branch is outdated. Make sure you update your fork branch before making any changes. 

Try updating your fork to resolve conflicts.

File: data/aadii0408.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Aaditya Champaneri || Aadi ",
+  "displayStatsPublic": true,
```
Comment (krshkun) [inline data/aadii0408.json:3]:
```suggestion
```

This is no longer a option 👍

**Review by krshkun — CHANGES_REQUESTED** [review]

File: data/aadii0408.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "type": "personal",
+  "bio": "Artist By Choice, Engineer By Default | Python Developer",
+  
+  "tags": [ "Python", "Machine Learning", "Open Source", "Marketing", "Communities"],
+  
+  "avatar": "https://github.com/aadii0408.png",
```
Comment (Hrishi5111998) [inline data/aadii0408.json:9]: hey @aadii0408 we don't need avatar in new update you can remove it

**Review by Hrishi5111998 — CHANGES_REQUESTED** [review]

File: data/aadii0408.json (hunk 2 above)
Comment (SaraJaoude) [reply data/aadii0408.json:8]:
```suggestion
```

**Review by Pradumnasaraf — APPROVED** [review]

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, there are some merge conflicts in the PR. Please fix them

**Review by devvsakib — CHANGES_REQUESTED** [review]
your pr has conflict

**Comment by SaraJaoude** [comment]
> Hey, there are some merge conflicts in the PR. Please fix them

@aadii0408 you might find this video helpful: https://youtu.be/eze7Y4Rxwtc

**Comment by SaraJaoude** [comment]
> your pr has conflict

@devvsakib it is best to check the conversation and not duplicate a comment as this can be confusing for the author.

**Comment by SaraJaoude** [comment]
Given the length of time and lack of activity by the author, I have resolved the conflicts and accepted their latest changes. 

@aadii0408 if you would like to make changes to your Profile and avoid conflicts, I would suggest you take a look at our Editing Guide in Docs: https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Pull request merged**